mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -123,17 +123,22 @@ def setup_and_check(self):  # {{{
 
         self.sectionName = 'streamfunctionMOC'
 
-        self.includeBolus = config.getboolean(self.sectionName, 'includeBolus')
-        if self.includeBolus:
-            # only add the bolus velocity if GM is enabled
-            self.includeBolus = self.namelist.getbool('config_use_standardgm')
-
-        self.variableList = ['timeMonthly_avg_normalVelocity',
-                             'timeMonthly_avg_vertVelocityTop']
-        if self.includeBolus:
-            self.variableList.extend(
-                    ['timeMonthly_avg_normalGMBolusVelocity',
-                     'timeMonthly_avg_vertGMBolusVelocityTop'])
+        if self.mocAnalysisMemberEnabled:
+            # I think the bolus velocity should always be included if GM is on
```

Comment:
I think the following should be changed so that it's no longer an option. If GM is on, we need to include the bolus velocity part.